modules/ROOT/pages/intro-overview.adoc

```diff
@@ -1,8 +1,12 @@
-= Introduction to Mule 4
+= Introduction to Mule 4 for Mule 3 Users
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
+If you are a user that already knows Mule 3 and recently migrated to Mule 4, review the following introduction overview to learn the main changes for this version. +
+
+If you are an experienced Mule 4 user, jump directly to our xref:index.adoc[Mule Runtime] documentation instead.
+
 Mule 4 simplifies the expression language and reduces management complexity so that you can speed up the on-ramping process and deliver applications faster than in Mule 3.
```

Comment:
```suggestion
Mule 4 simplifies the expression language and reduces management complexity so that you can ramp up and deliver applications faster than in Mule 3.
```